solution/parser/fr-parser/main.go

```diff
@@ -140,7 +140,7 @@ func start() error {
 		log.Info("[main] retrieving content for title ", title.Title)
 		parts := getPartsListFunc(ctx, title)
 		for _, part := range parts {
-			if err := processPartFunc(ctx, title.Title, part, existingDocs, config.SkipVersions, titles); err != nil {
+			if err := processPartFunc(ctx, title.Title, part, existingDocs, config.SkipFRDocuments, titles); err != nil {
```

Comment:
This method processPartFunc takes a lot of arguments. Generally best practice to have max of 3 arguments.  https://medium.com/@meeusdylan/go-reduce-function-parameters-19b785a87a59

Reply:
Happy to oblige on this! But I think it should be another tech debt ticket honestly, it's out of the scope of this fix and would take more significant refactoring to make all code compliant.

Reply:
Sounds good, please go ahead and create a ticket for this. And I will approve this pull request.